app/components/forms/session-speaker-form.js

```diff
@@ -12,6 +12,7 @@ export default Component.extend(FormMixin, {
 
   newSpeakerSelected : false,
   newSessionSelected : false,
+  aspectRatio : [2, 2],
```

Comment:
```suggestion
  aspectRatio : [1, 1],
```

Reply:
@iamareebjamal This should be 2:2, As ui medium image except an image for 300 px for non pixelation